pkg/debugapi/accounting_info.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package debugapi
+
+import (
+	"net/http"
+
+	"github.com/ethersphere/bee/pkg/bigint"
+	"github.com/ethersphere/bee/pkg/jsonhttp"
+)
+
+var (
+	errCantInfo = "Cannot get accounting infos"
+)
+
+type infoResponseArray struct {
+	InfoResponse []accountingInfoResponse `json:"infos"`
+}
+
+type accountingInfoResponse struct {
+	Peer                  string         `json:"peer"`
```

Comment:
since the map from the accounting package is already sorting entries by kv pairs (key is address, value is accounting data), you can omit this field